index.js

```diff
@@ -1,26 +1,142 @@
-const { question } = require("readline-sync");
-const { displayWordSoFar, isGameWon, isGameLost } = require("./gamelogic");
+const { question } = require('readline-sync');
+const { displayWordSoFar, isGameWon, isGameLost } = require('./gamelogic');
+let counter = 0;
 
 function game(word, guesses) {
-  console.log("Dit heb je tot nu toe geraden: ", guesses);
+  console.log('Dit heb je tot nu toe geraden: ', displayWordSoFar(word, guesses));
+  const letter = question('Raad een letter: ');
 
-  const letter = question("Raad een letter: ");
+  // Tel de foute antwoorden op
+  if (!word.includes(letter)) {
```

Comment:
Zie ook mijn comment bij het tonen van je galgje, voor de leesbaarheid van deze code en om encapsulatie (bepaalde toegang wel/niet verschaffen) te vergroten is het goed om een methode apart te schrijven van je index zodat de index alleen bepaalde functionaliteit laat zien. 

MAW: wat je doet is helemaal niet fout, maar je kunt het mooier maken door het van elkaar los te koppelen. :)